### ISSUE TITLE
Psychonauts

### ISSUE THREAD

**Issue by Firlaev-Hans** [ptitSeb/box86-compatibility-list#93]
### Description

Native Linux port, Steam version launched without steam with Goldberg Steam Emu. On an actual x86 PC, the game window captures the mouse and never lets go of it and even prevents Alt+Tab sometimes, which is annoying. However, with box86, the game window fails to capture the mouse at all, making it hard to move the camera and impossible to click around the settings menu (?)

### Tested on

Pine64 RockPro64 4GB (RK3399 with Panfrost)
TwisterOS

### Status
**Previous behavior:** 

None reported

**Current behavior:**

Runs but has problems with the mouse, performance is sub-par

### Screenshots

![Psychonauts](https://user-images.githubusercontent.com/61254500/117881695-d7858100-b2a9-11eb-97a3-fe64f69190b0.png)
Depending on what you are looking at, the performance jumps between 15 and 20 FPS

### OpenGL

3.1 (I think the game just needs 2.1)

**Comment by clort81** [comment]
A lot of emulated games have difficulty with mouse in windowed mode.  How is it in fullscreen?

**Comment by mothenjoyer69** [comment]
For anyone dropping by this in the future, the native GOG version starts out of the box on my Phytium D2000 system. The windowed mouse issue still exists and it makes it extremely difficult to enter fullscreen (@clort81 Unfortunately I can't test fullscreen for you right now as a result). 

The performance is a bit hit or miss, I get very odd CPU and GPU utilization and all-over-the-place framerates, lowest being 15fps.

**Comment by mingdug** [comment]
> For anyone dropping by this in the future, the native GOG version starts out of the box on my Phytium D2000 system. The windowed mouse issue still exists and it makes it extremely difficult to enter fullscreen (@clort81 Unfortunately I can't test fullscreen for you right now as a result).
> 
> The performance is a bit hit or miss, I get very odd CPU and GPU utilization and all-over-the-place framerates, lowest being 15fps.

Yeah I tried it myself with my OPi5 and if I just stand still for a bit it seems to get more stable, resolution and graphical settings don't affect FPS whatsoever for me whether from 1080p or 480p nor does switching everything from lowest to highest. Probably a bug with the emulator, the audio is perfect though.

**Comment by Dia2809** [comment]
Hey, i cant get it to work on my Powkiddy X55 with Rocknix. It has a rk3566 and i wanted to know which Box86 version yall were using. It crashes after the video for me
